baybe/parameters/validation.py

```diff
@@ -29,7 +29,9 @@ def validate_decorrelation(obj: Any, attribute: Any, value: float) -> None:
         lt(1.0)(obj, attribute, value)
 
 
-def validate_is_finite(obj: Any, _: Any, value: Sequence[float]) -> None:
+def validate_is_finite(  # noqa: DOC101, DOC103
```

Comment:
I'm not sure if I fully understand the reason for this PR. Didn't we need to add these execptions also earlier on because the linters complained otherwise!? 🤔

Reply:
The reason is written in the PR description, what part is unclear there?

Reply:
So I understand you are saying that before the change, things were not properly checked, because there was that missing `noqa` that you had to add after the change (which means that before the change, the linter didn't even bother about that line). But I also remember that we HAD TO put the DOC101 and DOC103 exceptions even before this change. So I am struggling to understand what exactly is different now 🤔 Or maybe it's just me who is confused :D

Reply:
Ok, I'll try to make it clear :D
At some point, we broke the pydoclint check. Don't know when, don't know why, don't know how. Probably when moving away from flake8. Still, it WAS present in the past which is why there was a point where we had to use the `noqa` stuff.
This PR basically just reactivates pydoclint now.

Reply:
Ok, I see. Well, then let's merge. 👍🏼

Reply:
@AVHopp  Feel free to move forward and merge, but it would be still really good to understand what happened. "don't know why, don't know how" is not a really good thing for a repo that we own :D You should be able to track the cause down with `git bisect` in 5mins. Just run it (see online how it works) and give in each iteration feedback as to whether it was working on the respective commit or not